docs/relations/sponsors.mdx

```diff
@@ -2,27 +2,48 @@
 sidebar_position: 0
 ---
 
+import ThemedImage from '@theme/ThemedImage'
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
 # Betaflight Sponsors
 
-<div className="grid xs:grid-cols-4 md:grid-cols-4 gap-4 mb-4 items-center justify-center h-full">
-  <a href="https://www.hqprop.com/" target="_blank" rel="noreferrer noopener" className="xl:col-span-1 col-span-2 no-effect">
-    <img src="/img/betaflight/sponsors/hqprop.svg" alt="HQProp" className="invert no-effect rounded-2xl w-4/5" />
-  </a>
-  <a href="https://https://www.radiomasterrc.com/" target="_blank" rel="noreferrer noopener" className="xl:col-span-1 col-span-2 no-effect">
-    <img src="/img/betaflight/sponsors/radiomaster-logo.jpg" alt="Radiomaster" className="no-effect rounded-2xl w-full" />
-  </a>
-  <a href="https://www.tititop.com/" target="_blank" rel="noreferrer noopener" className="xl:col-span-1 col-span-2 no-effect">
-    <img src="/img/betaflight/sponsors/dogcom.svg" alt="Radiomaster" className="no-effect rounded-2xl w-full" />
-  </a>
-  <a href="https://www.axisflying.com/" target="_blank" rel="noreferrer noopener" className="xl:col-span-1 col-span-2 no-effect">
-    <img src="/img/betaflight/sponsors/Axisflying_dark.svg" alt="AxisLogo" className="no-effect rounded-2xl w-full" />
+<div className="flex flex-col items-center backdrop-blur-md shadow-xl p-4 rounded-2xl bg-neutral-500/10 grid md:grid-cols-4 gap-4 mb-4">
+  <a href="https://www.hqprop.com/" target="_blank" rel="noreferrer noopener" className="flex flex-col items-center xl:col-span-1 col-span-2 no-effect">
+    <img
+      alt="HQProp"
+      className="no-effect w-4/5 themeInvert"
```

Comment:
HQprop now seems too small; changing `w-⅘` to `w-full`, balances it better with the others.

Reply:
Perhaps it was a caching error, but the image sizes now appear OK after rebooting and force-refreshing.
Please ignore the previous suggestion.